spec/board_spec.rb

```diff
@@ -0,0 +1,99 @@
+require './lib/board'
+
+describe Board do
+  let(:gameboard) { Board.new }
+  describe '#play_again' do
+    it 'should output an array of 9 dashes' do
+      expect(gameboard.play_again).to eq(%w[- - - - - - - - -])
+    end
+  end
+
+  describe '#horizontal_winner?(player)' do
+    it 'should return true when get a horizontal line of symbol X' do
+      gameboard.update(1, 'X')
+      gameboard.update(2, 'X')
+      gameboard.update(3, 'X')
+      expect(gameboard.horizontal_winner?('X')).to eq(true)
+    end
+    it 'should return false when not get a horizontal line of symbol X' do
+      gameboard.update(1, 'Y')
+      gameboard.update(2, 'X')
+      gameboard.update(3, 'Y')
+      expect(gameboard.horizontal_winner?('X')).to eq(false)
+    end
+  end
+
+  describe '#vertical_winner?(player)' do
+    it 'should return true when get a vertical line of symbol X' do
```

Comment:
* [ ] Currently, the test for vertical winner captures only one situation thus ```1,4,7``` very well, but is silent on position ```2,5,8``` and ```3,6,9```. Kindly add the situation whereby the winner is at position ```2,5,8```, and ```3,6,9``` to match the implementation of the method.